### PR TITLE
Tidying up some of the core OSCAL methods

### PR DESCRIPTION
**Pull request by anweiss**
Tidying up some of the core OSCAL methods. Accept `io.Writer` as appropriate. Removed CC0 Public Domain header from all source code files in favor of blanket `LICENSE` file at the root.

### CODE REVIEW

File: types/oscal/oscal.go [hunk 1]
```diff
@@ -183,30 +148,34 @@ func New(r io.Reader) (*OSCAL, error) {
...
 
-// RawXML ...
-func (o *OSCAL) RawXML(prettify bool) ([]byte, error) {
+// XML writes the OSCAL object as XML to the given writer
+func (o *OSCAL) XML(w io.Writer, prettify bool) error {
+	e := xml.NewEncoder(w)
 	if prettify {
-		return xml.MarshalIndent(o, "", "  ")
+		e.Indent("", "  ")
+		return e.Encode(o)
```
Comment (asadullah-yousuf-10p) [inline types/oscal/oscal.go]: This statement might not be needed as it’s already there at the end of the method (line 164)

File: types/oscal/oscal.go [hunk 2]
```diff
@@ -183,30 +148,34 @@ func New(r io.Reader) (*OSCAL, error) {
...
 
-// RawJSON ...
-func (o *OSCAL) RawJSON(prettify bool) ([]byte, error) {
+// JSON writes the OSCAL object as JSON to the given writer
+func (o *OSCAL) JSON(w io.Writer, prettify bool) error {
+	e := json.NewEncoder(w)
 	if prettify {
-		return json.MarshalIndent(o, "", "  ")
+		e.SetIndent("", "  ")
+		return e.Encode(o)
```
Comment (asadullah-yousuf-10p) [inline types/oscal/oscal.go]: Same change suggested as the one on line 161. 

Also, I see a lot of common code in this method and the xml one apart from the statement where an encoder is created. Do you think we can extract a new method and use that method here and above?

Comment (anweiss) [reply types/oscal/oscal.go]: Good catch, thanks.

**Comment by anweiss** [comment]
@asadullah-yousuf-10p updated per your suggestions. Let me know what you think. Thanks.

**Pull request merged**